tests/NLog.UnitTests/Targets/FileTargetTests.cs

```diff
@@ -562,6 +562,71 @@ public void DeleteArchiveFilesByDate()
             }
         }
 
+        // Test commented out as it takes 5 minutes to run
+        /*
+        [Fact]
+        public void DeleteArchiveFilesByDateWithDateName()
+        {
+            var tempPath = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
+            var tempFile = Path.Combine(tempPath, "${date:format=yyyyMMddHHmm}.txt");

```

Comment:
You can use the format `yyyMMddHHmmss` to have new file created every second, so this test would require 5 seconds to run instead of 5 minutes.